ModularTegustation/harvestable_gardens.dm

```diff
@@ -0,0 +1,161 @@
+	regrowth_time_low = 20 MINUTES
+	regrowth_time_high = 40 MINUTES
+	num_sprites = 1
+	light_power = 0.5
+	light_range = 1
+	needs_sharp_harvest = FALSE
+
+/obj/structure/flora/ash/garden/arid
+	name = "sandy garden"
+	desc = "Beneath a bluff of soft silicate, a sheltered grove slumbers."
+	icon_state = "gardenarid"
+	harvested_name = "sandy garden"
+	harvested_desc = "Beneath a bluff of soft silicate, a sheltered grove slumbered. Some desert wanderer seems to have picked it clean."
+	harvest = /obj/effect/spawner/lootdrop/garden/arid
+	harvest_amount_high = 1
+	harvest_amount_low = 1
+	harvest_message_low = "You brush sand away from a verdant prize, nestled in the leaves."
+	harvest_message_med = "You brush sand away from a verdant prize, nestled in the leaves."
+	harvest_message_high = "You brush sand away from a verdant prize, nestled in the leaves."
+
+/obj/structure/flora/ash/garden/frigid
+	name = "chilly garden"
+	desc = "A delicate layer of frost covers hardy brush."
+	icon_state = "gardencold"
+	harvested_name = "chilly garden"
+	harvested_desc = "A delicate layer of frost covers hardy brush. Someone came with the blizzard, and left with any prize this might contain."
+	harvest = /obj/effect/spawner/lootdrop/garden/cold
+	harvest_amount_high = 1
+	harvest_amount_low = 1
+	harvest_message_low = "You unearth a snow-covered treat."
+	harvest_message_med = "You unearth a snow-covered treat."
+	harvest_message_high = "You unearth a snow-covered treat."
+
+/obj/structure/flora/ash/garden/waste
+	name = "sickly garden"
+	desc = "Polluted water wells up from the cracked earth, feeding a patch of something curious."
+	icon_state = "gardensick"
+	harvested_name = "sickly garden"
+	harvested_desc = "Polluted water wells up from the cracked earth, where it once fed a patch of something curious. Now only wilted leaves remain."
+	harvest = /obj/effect/spawner/lootdrop/garden/sick
+	harvest_amount_high = 1
+	harvest_amount_low = 1
+	harvest_message_low = "You pry something odd from the poisoned soil."
+	harvest_message_med = "You pry something odd from the poisoned soil."
+	harvest_message_high = "You pry something odd from the poisoned soil."
+
+/obj/structure/flora/ash/garden/seaweed //yea, i code :)
+	name = "seaweed patch"
+	gender = NEUTER
+	desc = "A patch of seaweed, floating on the surface of the water"
+	icon_state = "seaweed"
+	harvested_name = "seaweed patch"
+	harvested_desc = "A patch of seaweed, floating on the surface of the water. It seems someone has already searched through this"
+	harvest = /obj/item/food/seaweed
+	harvest_amount_high = 2
+	harvest_amount_low = 1
+	harvest_message_low = "You discover some edible weeds within the patch."
+	harvest_message_med = "You discover some edible weeds within the patch."
+	harvest_message_high = "You discover some edible weeds within the patch."
+
+/obj/effect/spawner/lootdrop/garden
+	name = "lush garden seeder"
+	lootcount = 3
+	var/list/plant = list(
```

Comment:
could use only being a single layer of indentation on the list's items, instead of 2 that it currently is

Reply:
I didnt even notice that they were indented weirdly.